### PR TITLE
doc(MigrationTool) : Removing upgrade from the documentation

### PR DESCRIPTION
**Pull request by vbotnari12**
Removing "upgrade" from the documentation of Migration Tool, because upgrade it's subscription change, not version change.

### CODE REVIEW

File: modules/ROOT/pages/migrate-from-an-earlier-version-of-bonita-bpm.adoc [hunk 1]
```diff
@@ -36,18 +36,18 @@ You are recommended not to start 7.0.0 after you migrate to it, but to proceed i
 ====
 
 *RDBMS requirements:*
-The version targeted may not support the version of the database that is being migrated. You may then need to upgrade the version of your database prior to running the migration tool.
+The version targeted may not support the version of the database that is being migrated. You may then need to update the version of your database prior to running the migration tool.
```
Comment (educhastenier) [inline modules/ROOT/pages/migrate-from-an-earlier-version-of-bonita-bpm.adoc]: @vbotnari12 We should check if database vendors use the term update of upgrade of migrate for the Database server "mise à jour"...

File: modules/ROOT/pages/migrate-from-an-earlier-version-of-bonita-bpm.adoc [hunk 2]
```diff
@@ -36,18 +36,18 @@ You are recommended not to start 7.0.0 after you migrate to it, but to proceed i
...
 
 * Please check the xref:hardware-and-software-requirements.adoc[database requirements].
-* If you need to upgrade your database:
+* If you need to update your database:
```
Comment (educhastenier) [inline modules/ROOT/pages/migrate-from-an-earlier-version-of-bonita-bpm.adoc]: same remark

**Comment by educhastenier** [comment]
@vbotnari12 ❓ Is it normal you make this PR on a 2021.1 base?

File: modules/ROOT/pages/migrate-from-an-earlier-version-of-bonita-bpm.adoc (hunk 1 above)
Comment (vbotnari12) [reply modules/ROOT/pages/migrate-from-an-earlier-version-of-bonita-bpm.adoc]: nice catch. AWS uses upgrade, Oracle uses both - upgrade and migration.

**Pull request closed without merging**